std/experimental/allocator/building_blocks/allocator_list.d

```diff
@@ -823,7 +830,10 @@ version (Posix) @system unittest
     import std.algorithm.comparison : max;
     import std.typecons : Ternary;
 
-    enum pageSize = 4096;
+    version (AppleARM)
+        enum pageSize = 16384;
+    else
+        enum pageSize = 4096;
```

Comment:
Maybe define this at the top of the module for to avoid repetition?

Reply:
Done.